specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/AnomalyDetector.json

```diff
@@ -106,41 +106,1303 @@
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Changepoint example": {
+            "$ref": "./examples/ChangePointDetectOnTimestamp.json"
+          }
+        }
+      }
+    },
+    "/timeseries/{timeSeriesId}/label": {
+      "post": {
+        "summary": "Label API is used for users to label detection status of a certain time stamp of a time series",
+        "description": "This operation is used for users to label Anomaly or ChangePoint states of a certain time stamp, these label will be used for regenerate detection model",
+        "operationId": "TimeSeries_Label",
+        "parameters": [
+          {
+            "$ref": "#/parameters/timeSeriesId"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "description": "In Label request, user can set Anomaly|ChangePoint state (true, false, unknown) for a time range",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/LabelRequest"
+            }
+          }
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful operation."
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/APIError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Label example": {
+            "$ref": "./examples/Label.json"
+          }
+        }
+      }
+    },
+    "/timeseries/{timeSeriesId}/query": {
+      "post": {
+        "description": "Query timeseries with required field in each timestamp.",
+        "operationId": "TimeSeries_Query",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/timeSeriesId"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "description": "Request body for querying timeseries.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/TimeSeriesQueryRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful operation.",
+            "schema": {
+              "$ref": "#/definitions/TimeSeriesQueryResponse"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/APIError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Query Timeseries example": {
+            "$ref": "./examples/TimeSeriesQuery.json"
+          }
+        }
+      }
+    },
+    "/timeseries/{timeSeriesId}/timeseriesgroups": {
```

Comment:
Combine getting group interface with the time series creating one, for example /anomalydetector/v1.0/timeseries/{timeSeriesId}? timeseriesgroups=true